app/models/submission.rb

```diff
@@ -26,4 +27,12 @@ def student_names
       end.join(' & ')
     end
   end
+
+  def feedback_provider
```

Comment:
This method seems more appropriate for a view helper as it's providing what text to display for the provider column.